### PR TITLE
Default GitIgnore Override

### LINKED ISSUE #112: Override gitignore
For now the generated gitignore is for java version so we can override for add dotnet version

### PR DESCRIPTION
**Pull request by romain325**
Override default gitignore for a DotNetCore adapted version
Fix #112

The value for the gitignore doesn't perfectly fit
Need Review to make an efficient one

### CODE REVIEW

**Comment by nicolas63** [comment]
you can look this file : (https://raw.githubusercontent.com/jhipster/jhipster-net-sample-app-template/master/.gitignore) 
you need this part 
```
#Angular
**/wwwroot/
**/ClientApp/dist/
```

You can also delete the gitignore in https://github.com/jhipster/jhipster-dotnetcore/blob/master/generators/common/index.js

**Comment by romain325** [comment]
I added different adapted .gitignore and removed the gitignore default in the index.js, I also noticed that the README.md was also copied from default and then overwritted by the dotnet specific 
I removed it and tested it, nothing changed --> need an approval to be sure i haven't done anything wrong

File: generators/common/templates/dotnetcore/gitignore.ejs [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+###########
+# Vs & vscode
+###########
+.vs
+.vscode
+
+#Angular
+**/wwwroot/
+**/ClientApp/dist/
```
Comment (nicolas63) [inline generators/common/templates/dotnetcore/gitignore.ejs:29]: duplicated lines

File: generators/common/templates/dotnetcore/gitignore.ejs [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+_Pvt_Extensions
+
+# Paket dependency manager
+.paket/paket.exe
+
+# FAKE - F# Make
+.fake/
+
+#IntelliJ
+.idea/
```
Comment (nicolas63) [inline generators/common/templates/dotnetcore/gitignore.ejs:289]: duplicated line

File: generators/common/templates/dotnetcore/gitignore.ejs [hunk 3]
```diff
@@ -0,0 +1,293 @@
...
+ClientBin/
+~$*
+*~
+*.dbmdl
+*.dbproj.schemaview
+*.pfx
+*.publishsettings
+orleans.codegen.cs
+
+node_modules/
```
Comment (nicolas63) [inline generators/common/templates/dotnetcore/gitignore.ejs:237]: duplicated line

**Comment by nicolas63** [comment]
Looks good to me
Congratulation for your first PR in this project :clap:
Can you remove draft mode ?

**Comment by romain325** [comment]
Draft mode disabled!

**Comment by nicolas63** [comment]
merged, thank you @romain325

**Pull request merged**